pkg/api/types/image_types.go

```diff
@@ -193,6 +198,8 @@ type ImagePullOptions struct {
 	Quiet bool
 	// multiaddr of IPFS API (default uses $IPFS_PATH env variable if defined or local directory ~/.ipfs)
 	IPFSAddress string
+	// Flags to pass into remote snapshotters
+	RFlags *RemoteSnapshotterFlags
```

Comment:
I don't think we need a pointer here.

Reply:
I think in general we don't need pointer for these flag types (they're not that large)